libgomp/affinity-fmt.c:330:25: error: initialization discards ‘const’ qualifier from pointer target type [-Werror=discarded-qualifiers]

--- ./libgomp/affinity-fmt.c.vanilla	2026-01-24 14:44:57.698092512 +0100
+++ ./libgomp/affinity-fmt.c	2026-01-24 14:45:09.690189316 +0100
@@ -327,7 +327,7 @@
 	      }
 	  if (c == '{')
 	    {
-	      char *q = strchr (p + 1, '}');
+	      const char *q = strchr (p + 1, '}');
 	      if (q)
 		gomp_fatal ("unsupported long type name '%.*s' in affinity "
 			    "format", (int) (q - (p + 1)), p + 1);
